.travis.yml

```diff
@@ -15,7 +15,7 @@ install:
 script:
   - python all_tests.py --coverage
 after_success:
-  - coveralls
+  - coveralls --rcfile=.coveragerc
```

Comment:
We did this in all AIMMO repos now. The coverage was taking tests and eggs into account.